pilot/pkg/model/push_context.go

```diff
@@ -72,6 +72,8 @@ type PushContext struct {
 	namespaceExportedDestRules map[string]*processedDestRules
 	allExportedDestRules       *processedDestRules
 
+	configVersion string
```

Comment:
A bit confusing to have a Version and configVersion. If we need both can they be next to each other with comments clearly explaining the difference?

Reply:
I'd be happy to add that, as soon as I understand version... do you know what version it is referring to?

Reply:
I am not sure, it looks like you are adding both?

Reply:
Oh. Right. My bad.  will remove configVersion